ml-agents/mlagents/trainers/learn.py

```diff
@@ -49,6 +50,23 @@ def parse_command_line(argv: Optional[List[str]] = None) -> RunOptions:
     return RunOptions.from_argparse(args)
 
 
+def _Get_checkpoint_name(
+    behavior_name: str, checkpoint_list: Optional[Dict[str, str]]
```

Comment:
argument is called checkpoint_list but it is a mapping from str to str. Please rename argument